DuckDuckGo/TabViewController.swift

```diff
@@ -71,38 +71,38 @@ class TabViewController: UIViewController {
     private(set) var webView: WKWebView!
     private lazy var appRatingPrompt: AppRatingPrompt = AppRatingPrompt()
     private weak var privacyController: PrivacyProtectionController?
-    
+
     private(set) lazy var appUrls: AppUrls = AppUrls()
     private var storageCache: StorageCache = AppDependencyProvider.shared.storageCache.current
     private lazy var appSettings = AppDependencyProvider.shared.appSettings
-    
+
     internal lazy var featureFlagger = AppDependencyProvider.shared.featureFlagger
     private lazy var featureFlaggerInternalUserDecider = AppDependencyProvider.shared.featureFlaggerInternalUserDecider
-    
+
     lazy var bookmarksManager = BookmarksManager()
 
     private(set) var urlToSiteRating: [URL: SiteRating] = [:]
     private(set) var siteRating: SiteRating?
     private(set) var tabModel: Tab
-    
+
     private let requeryLogic = RequeryLogic()
-    
+
     private static let tld = AppDependencyProvider.shared.storageCache.current.tld
     private let adClickAttributionDetection = ContentBlocking.makeAdClickAttributionDetection(tld: tld)
     let adClickAttributionLogic = ContentBlocking.makeAdClickAttributionLogic(tld: tld)
-    
+
     private var httpsForced: Bool = false
     private var lastUpgradedURL: URL?
     private var lastError: Error?
     private var shouldReloadOnError = false
     private var failingUrls = Set<String>()
-    
+
     private var trackerNetworksDetectedOnPage = Set<String>()
     private var pageHasTrackers = false
-    
+
     private var detectedLoginURL: URL?
     private var preserveLoginsWorker: PreserveLoginsWorker?
-    
+
     private var trackersInfoWorkItem: DispatchWorkItem?
```

Comment:
Any reason for this? If not maybe would be better to discard this chunk

Reply:
Xcode applies no leading spaces formatting to lines that were touched, I remember the onboarding task with code styling requirement to have this Xcode setting to be ON, I think we should put this on the Weekly Agenda and enforce this rule once again because I believe it‘s not strongly followed